[2025-10-23T20:08:31.224+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-23T20:08:31.255+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v03.get_name manual__2025-10-23T20:08:28.812181+00:00 [queued]>
[2025-10-23T20:08:31.267+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v03.get_name manual__2025-10-23T20:08:28.812181+00:00 [queued]>
[2025-10-23T20:08:31.269+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-10-23T20:08:31.290+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2025-10-23 20:08:28.812181+00:00
[2025-10-23T20:08:31.306+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v03', 'get_name', 'manual__2025-10-23T20:08:28.812181+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp4g43baqs']
[2025-10-23T20:08:31.305+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1719) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-23T20:08:31.310+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask get_name
[2025-10-23T20:08:31.310+0000] {standard_task_runner.py:72} INFO - Started process 1720 to run task
[2025-10-23T20:08:31.422+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_v03.get_name manual__2025-10-23T20:08:28.812181+00:00 [running]> on host c65f76f8d89c
[2025-10-23T20:08:31.560+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v03' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T20:08:28.812181+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T20:08:28.812181+00:00'
[2025-10-23T20:08:31.561+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-23T20:08:31.599+0000] {python.py:240} INFO - Done. Returned value was: Olovuddin
[2025-10-23T20:08:31.647+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-23T20:08:31.648+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v03, task_id=get_name, run_id=manual__2025-10-23T20:08:28.812181+00:00, execution_date=20251023T200828, start_date=20251023T200831, end_date=20251023T200831
[2025-10-23T20:08:31.738+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-23T20:08:31.763+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-23T20:08:31.767+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
